hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestTable.java

```diff
@@ -268,16 +292,16 @@ public boolean baseFileExists(String partition, String instantTime, String fileI
     }
   }
 
-  public Path getPartitionPath(String partition) {
-    return new Path(Paths.get(basePath, partition).toUri());
-  }
-
-  public String getBaseFileNameById(String fileId) {
-    return baseFileName(currentInstantTime, fileId);
+  public boolean logFilesExist(String partition, String instantTime, String fileId, int... versions) {
+    return Arrays.stream(versions).allMatch(v -> logFileExists(partition, instantTime, fileId, v));
   }
 
-  public Path getBaseFilePath(String partition, String fileId) {
-    return new Path(Paths.get(basePath, partition, getBaseFileNameById(fileId)).toUri());
+  public boolean logFileExists(String partition, String instantTime, String fileId, int version) {
+    try {
+      return fs.exists(new Path(Paths.get(basePath, partition, logFileName(instantTime, fileId, version)).toString()));
+    } catch (IOException e) {
+      throw new HoodieTestTableException(e);
+    }
```

Comment:
just reordering the APIs. API order is sth like inflight commit, commit, basefile, logfile,